dts/arm/stm32_min_dev.dts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2017, embedjournal.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32f103Xb.dtsi>
+
+/ {
+	model = "STM32 Minimum Development Board";
+	compatible = "st,stm32_min_dev", "st,stm32f103rb";
```

Comment:
I'm not sure if this is correct as technically, this is stm32f103rb.
Anyway, let's keep it that way as this is exact same a part from memory footprint, which is already handled